setup.py

```diff
@@ -15,7 +15,8 @@
     'numpy>=1.18.0,<2',
     'pandas>=1.1,<1.1.5',
     'scikit-learn>=0.23,<1',
-    'scipy>=1.4.1,<2',
+    'scipy>=1.4.1,<1.5',
```

Comment:
Why are we capping `scipy` at 1.5? I believe this will create conflicts with our other repos, since they support up to `2`.

Reply:
I reverted it, I changed it while I was trying to fix the dependencies  and forgot to roll it back.